api-client/src/main/java/com/xing/api/RealCallSpec.java

```diff
@@ -142,6 +144,16 @@ public void onResponse(Call call, okhttp3.Response rawResponse) {
     }
 
     @Override
+    public Single<RT> singleStream() {
+        return stream().toSingle();
+    }
+
+    @Override
+    public Completable completableStream() {
+        return stream().toCompletable();
```

Comment:
Can you annotate this with `@Experimental` just for now... Because, I'm thinking to try to drop the parsing completely...